lib/index.d.ts

```diff
@@ -23,7 +23,7 @@ declare namespace Stardog {
         export interface Body {
             status: string;
             statusText: string;
-            result: object | string | boolean | null;
+            result: Object | string | boolean | null;
```

Comment:
it's `object, lowercase "o". We had this change before.


> Don’t ever use the types Number, String, Boolean, or Object. These types refer to non-primitive boxed objects that are almost never used appropriately in JavaScript code.

-Taken from [here](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html)